### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by lucyferrabee**

### CODE REVIEW

File: .gitignore.txt [hunk 1]
```diff
@@ -0,0 +1,168 @@
+
```
Comment (mporam) [inline .gitignore.txt]: this file should not have .txt on the end, which is why the .DS_Store files are in github. remove the .DS_Store files (ask if you need me to show you how), then rename this file to not have .txt on the end

File: index.html [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+		<link href="normalize.css" rel="stylesheet" type="text/css">
+		<link href="style.css" rel="stylesheet" type="text/css">
+		<title>Lucy Ferrabee</title>
+		<meta charset="UTF-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	</head>
+	<body>
+		<div class="top-bar-container">
+			<section class="top-bar">
+				<h1 class="logo"><a>Lucy Ferrabee</a></h1>
```
Comment (mporam) [inline index.html]: no href on the anchor?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+					<a href="#projects">Projects</a>
+					<a href="#contact">Contact</a>
+					<a href="#">CV</a>
+				</nav>
+			</section>
+		</div>
+		<section class="top-grid">
+			<section class="about">
+				<h2>Hi, I'm Lucy. :)</h2>
+				<ul><li>Aspiring Full Stack Developer</li>
```
Comment (mporam) [inline index.html]: put the `<li>` on its own line:
```
<ul>
    <li>...</li>
</ul>
```

File: index.html [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+			</section>
+			<section class="photo">
+				<img class="profile-photo" src="pp.jpeg">
+			</section>
+			<div class="socials">
+				<a href="https://www.linkedin.com/in/lucy-ferrabee/"><img class="linkedin" src="LI-In-Bug.png"></a>
+				<a href="https://github.com/lucyferrabee"><img class="github" src="GitHub-Mark-64px.png"></a>
+				<a href="mailto:lucy.ferrabee@hotmail.com"><img class="email" src="email_logo.png"></a>
+			</div>
+			<button class="my-projects"><a href="#projects">My Projects &#8595;</a>
```
Comment (mporam) [inline index.html]: why a button and an anchor?

File: index.html [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+			</button>
+		</section>
+  
+  <!--Portfolio Section-->
+		<div class="projects">
+			<div class="col">
+				<h3 id="projects">Project 1 Title</h3>
+				<a href="#"><img class="project-image" src="http://placekitten.com/300/300"></a>
+				<div><a href="#">Github</a></div>
```
Comment (mporam) [inline index.html]: you need 2 links for each project, one to the github and one for the live version of the project. we will host all your projects for you :)

File: index.html [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+			<img class="linkedin-photo" src="lipp.png">
+			<div class="socials2">
+				<a href="https://www.linkedin.com/in/lucy-ferrabee/"><img class="linkedin2" src="LI-In-Bug.png"></a>
+				<a href="https://github.com/lucyferrabee"><img class="github2" src="GitHub-Mark-64px.png"></a>
+				<a href="mailto:lucy.ferrabee@hotmail.com"><img class="email2" src="email_logo.png"></a>
+			</div>
+			<div class="letstalk"><p>Get in touch!</p></div>
+		</section>
+	</body>
+<footer>&copy; Copyright 2021 Lucy Ferrabee</footer>
```
Comment (mporam) [inline index.html]: footer should be inside the body tag. anything that displays on the page goes inside the body

File: style.css [hunk 7]
```diff
@@ -0,0 +1,304 @@
+* {
+box-size: border-box;
```
Comment (mporam) [inline style.css]: incorrect indentation

File: style.css [hunk 8]
```diff
@@ -0,0 +1,304 @@
...
+nav a {
+	font-size: 1.5rem;
+}
+
+nav a:hover {
+	color: black;
+}
+
+nav {
+	padding: 30px 30px 0px 0px;
```
Comment (mporam) [inline style.css]: dont need `px` on 0 values

File: style.css [hunk 9]
```diff
@@ -0,0 +1,304 @@
...
+	position: fixed;
+	top: 0;
+	right: 0;
+	z-index:99;
```
Comment (mporam) [inline style.css]: missing space after the `:`

File: style.css [hunk 10]
```diff
@@ -0,0 +1,304 @@
...
+	display: flex;
+	justify-content: space-between;
+	width: 100%;
+	position: fixed;
+	background-color: #64b5ae;
+	font-size: 1rem;
+}
+.top-bar-container {
+	background-color: #64b5ae;
+	z-index: 99;
```
Comment (mporam) [inline style.css]: this doesnt do anything. can only z-index non-static elements

File: style.css [hunk 11]
```diff
@@ -0,0 +1,304 @@
...
+body {
+	background-color: #64b5ae;
+	background-image: url("squiggle.png");
+	background-repeat: repeat-y;
+	font-size: 62.5%;
+	font-family: 'Roboto Mono', monospace;
+	color: black;
+}
+
+.top-bar {
```
Comment (mporam) [inline style.css:14]: this is the one that needs the z-index

File: style.css [hunk 12]
```diff
@@ -0,0 +1,304 @@
...
+	width: 40px;
+	height: 40px;
+}
+
+.socials {
+	display: flex;
+	flex-direction: column;
+	padding: 50px;
+	position: fixed;
+	z-index:98;
```
Comment (mporam) [inline style.css]: missing space before `:`

File: style.css [hunk 13]
```diff
@@ -0,0 +1,304 @@
...
+.email {
+	position: fixed;
+	top: 250px;
+	right: 50px;
+	width: 100%;
+}
+
+.top-bar a {
+	text-decoration: none;
+	color: white;
```
Comment (mporam) [inline style.css]: hex colors please

File: style.css [hunk 14]
```diff
@@ -0,0 +1,304 @@
...
+	border-radius: 45px;
+	box-shadow: 0px 8px 15px rgba(0, 0, 0, 0.1);
+	transition: all 0.3s ease 0s;
+	cursor: pointer;
+	outline: none;
+  	position: relative;
+	justify-content: center;
+  	top: 5px;
+	text-decoration: none;
+	color: black;
```
Comment (mporam) [inline style.css]: you have 2 color properties in here, remove this one

File: style.css [hunk 15]
```diff
@@ -0,0 +1,304 @@
...
+	width: 30%;
+
+}
+
+.my-projects a {
+	text-decoration:none;
```
Comment (mporam) [inline style.css]: missing space after the `:`

File: style.css [hunk 16]
```diff
@@ -0,0 +1,304 @@
...
+	text-decoration: none;
+}
+
+.projects img {
+	width: 350px;
+	height: 300px;
+}
+
+.projects {
+	display: inline-flex;
```
Comment (mporam) [inline style.css]: is there a reason you have used `inline-flex` instead of just `flex`?

File: style.css [hunk 17]
```diff
@@ -0,0 +1,304 @@
...
+	display: flex;
+	flex-direction: row;
+	width: 100px;
+	padding: 50px;
+	position: relative;
+	top: 30px;
+}
+
+.letstalk p {
+
```
Comment (mporam) [inline style.css]: remove empty line

File: style.css [hunk 18]
```diff
@@ -0,0 +1,304 @@
...
+	font-size: 4rem;
+	font-weight: bold;
+
```
Comment (mporam) [inline style.css]: remove empty line

File: style.css [hunk 19]
```diff
@@ -0,0 +1,304 @@
...
+	color: black;
```
Comment (mporam) [inline style.css]: hex

File: style.css [hunk 20]
```diff
@@ -0,0 +1,304 @@
...
+}
+
+footer {
+	padding-left: 60px;
+}
+
```
Comment (mporam) [inline style.css:216]: remove empty line

File: style.css [hunk 21]
```diff
@@ -0,0 +1,304 @@
...
+
+
+@media only screen and (max-width: 1274px) {
+
```
Comment (mporam) [inline style.css]: remove empty line

File: style.css [hunk 22]
```diff
@@ -0,0 +1,304 @@
...
+	.profile-photo {
+		width: 400px;
+		height: 400px;
+		padding: 0px 0px 0px 80px;
+	}
+}
+
+@media only screen and (max-width: 1074px) {
+
```
Comment (mporam) [inline style.css:227]: remove empty line

File: style.css [hunk 23]
```diff
@@ -0,0 +1,304 @@
...
+	
+	.linkedin-photo {
+		height: 125px;
+		width: 150px;
+	}
+  
+	.letstalk p {
+		display: none;
+	}
+
```
Comment (mporam) [inline style.css]: remove empty line

File: style.css [hunk 24]
```diff
@@ -0,0 +1,304 @@
...
+		width: 150px;
+	}
+  
+	.letstalk p {
+		display: none;
+	}
+	
+	
+}
+
```
Comment (mporam) [inline style.css:256]: remove empty line

**Review by mporam — CHANGES_REQUESTED** [review]

File: index.html (hunk 5 above)
Comment (lucyferrabee) [reply index.html]: There's a link on each project image, the plan was to be able to click to the live site by clicking on the image, or the github source underneath, but if it's not intuitively clear i'll change it to have two links

**Review by mporam — APPROVED** [review]

**Pull request merged**